addOns/help/src/main/javahelp/contents/releases/2.16.0.html

```diff
@@ -9,33 +9,63 @@
 <BODY>
 <H1>Release 2.16.0</H1>
 
-This is a bug fix and enhancement release.
+This is a bug fix and enhancement release. Look out for new 
+<a href="https://www.zaproxy.org/blog/">Blog Posts</a> and <a href="https://www.zaproxy.org/videos/">Videos</a> 
+which will cover some of these new features in much more depth in the coming days and weeks.
+
 <br>
 These release notes do not include all of the changes included in add-ons updated since 2.15.0.
 <p>
 Some of the more significant enhancements include:
 
 <H3>Update to a Minimum of Java 17</H3>
 
-TODO add details.
+ZAP now requires a minimum of Java 17 to run. This allows us to use more modern Java features in the ZAP codebase.
 
-<H3>Menu Items Restructured</H3>
+<H3>Detachable Tabs</H3>
 
-The desktop context sensitive menu items have been reordered, and grouped in a more logical way.
-This should make it much easier to find the menu item you want, when you want it.
+You can now detach any of the ZAP desktop tabs so that they will appear in their own windows.
+This means that you can see more than one tab in the group at the same time, and allows you to distribute those windows
+across multiple monitors. 
+Just right click any tab to see the 'detach' option and close the tab window to return it to the main ZAP window.
 
-<H3>Detachable Tabs</H3>
+<H3>Standard Policy Definitions</H3>
+
+We now have a standard set of active scan policy definitions which we recommend for use in specific situations.
+Using these definitions will just enable the most suitable active scan rules. This will speed up your active scans
+and will reduce noise. Some of the standard active scan rules are really targetted at pentesters, and these can waste 
+developers time.
+<p>
+For more details see the new <a href="https://www.zaproxy.org/docs/desktop/addons/scan-policies/">Scan Policies</a> add-on.
```

Comment:
No hyphen

Reply:
No, the link is based on the TOC text entry and spaces are converted to hyphens.

Reply:
The link does need fix in the add-on manifest.

Reply:
Oh okay, I'd checked the manifest that's why I commented :wink: